web3/pm.py

```diff
@@ -463,12 +476,36 @@ def release_package(
         to be the registry owner.
 
         * Parameters:
-            * ``package_name``: Must be a valid package name.
-            * ``version``: Must be a valid package version.
-            * ``manifest_uri``: Must be a valid manifest URI.
-        """
-        validate_package_name(package_name)
-        validate_package_version(version)
+            * ``package_name``: Must be a valid package name, matching the given manifest.
+            * ``version``: Must be a valid package version, matching the given manifest.
+            * ``manifest_uri``: Must be a valid content-addressed URI. Currently, only IPFS
+            and Github content-addressed URIs are supported.
+        """
+        if (
+            not is_ipfs_uri(manifest_uri) and
+            not is_valid_content_addressed_github_uri(manifest_uri)
+        ):
+            raise PMError(
```

Comment:
Seems like this should be some sort of `ValidationError` type error.  The base library error is normally not used directly, but rather designed to be a way for someone to *catch any library error from `py-ethpm`*